data/tools/Android.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Thank you for your effort, but creating a new category is not that easy, there are many files to crate (menu entries, scripts, etc.)